client/components/containers/Review/Review.js

```diff
@@ -11,6 +13,7 @@ class Review extends Component {
       topicId: '',
     };
     this.handleStarClick = this.handleStarClick.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
   }
 
   handleStarClick(e, topicId) {

```

Comment:
What is this function doing?

Reply:
that Review component isn't done but the handleSubmit button is now sending a req to the server and updating a partner's proficiencyRating as well as changing the user's status to 'completed'